localisation/00_wc_vanilla_replacements.csv

```diff
@@ -255,4 +255,16 @@ bloodthirsty_mass_sacrifice_bonus_3_desc;By appeasing [Root.Religion.GetHighGodN
 EVTDESCAHF23464;[From.GetTitledFirstName] has performed a great sacrifice, slaughtering hundreds of our own people to garner the favor of [From.Religion.GetHighGodName] during this war. [From.GetHerHisCap] fanatic obsession with sacrifices will doom us all.;;;;;;;;;;;;;x
 EVTDESCAHF23465;[From.GetTitledFirstName] has performed a great sacrifice, slaughtering hundreds of [Root.GetHerHis] own people to garner the favor of [From.Religion.GetHighGodName] during this war. [From.GetSheHeCap] hopes that the Blood God will now grant [From.GetHerHis] armies the strength necessary to defeat my warriors.;;;;;;;;;;;;;x
-EVTOPTAHF23465;What if [From.Religion.GetHighGodName] listened to [From.GetHerHim]?;;;;;;;;;;;;;x
\ No newline at end of file
+EVTOPTAHF23465;What if [From.Religion.GetHighGodName] listened to [From.GetHerHim]?;;;;;;;;;;;;;x
+MNM.7500.d;As [target_localization_sacrifice.GetTitledFirstName] is bound to the altar [target_localization_sacrifice.GetSheHe] appears to be mumbling prayers through [target_localization_sacrifice.GetHerHis] gag. [target_localization_sacrifice.GetSheHeCap] is sweating profusely, either due to the heat from the nearby pyres or from fear. [target_localization_sacrifice.GetHerHisCap] holy vestments hang in tatters already, exposing [target_localization_sacrifice.GetHerHis] chest. As the appointed hour draws near the chanting of your brethren builds to a crescendo until, finally, it is time.;Alors que [target_localization_sacrifice.Get_le_TitledFirstName] est ligot�[target_localization_sacrifice.Get_E] sur l'autel, [target_localization_sacrifice.GetIlElle] semble murmurer des pri�res � travers son b�illon. [target_localization_sacrifice.GetIlElleCap] transpire abondamment, soit � cause de la chaleur des b�chers voisins, soit de peur. Ses v�tements sacr�s pendent d�j� en lambeaux, exposant sa poitrine. Alors que l'heure pr�vue s'approche, le chant de vos fr�res s'accentue crescendo jusqu'� ce que, enfin, le moment soit venu.;[target_localization_sacrifice.GetTitledFirstName] ist an den Altar gefesselt und [target_localization_sacrifice.GetSheHe] scheint durch den Knebel Gebete zu murmeln. [target_localization_sacrifice.GetHerHisCap] K�rper ist schwei�nass, sei es aufgrund der Hitze der umliegenden Scheiterhaufen oder aus Angst. Das heilige Gewand h�ngt [target_localization_sacrifice.Getihmihr] in Fetzen von der Brust. Als die angesetzte Stunde n�her r�ckt, w�chst der Gesang Eurer Br�der zu einem Crescendo heran, bis es endlich so weit ist.;;Han atado a [target_localization_sacrifice.GetTitledFirstName] al altar. Crees o�r sus s�plicas a trav�s de la mordaza. Est� sudando como un animal por el miedo o por las piras encendidas a su alrededor. Sus santas vestiduras han quedado reducidas a jirones y su pecho ha quedado al descubierto. Se acerca la hora prevista y los c�nticos de los miembros de la orden llegan a un punto �lgido. Es el momento.;;;;;;;;;x
```

Comment:
Just an idea. Could you copypaste vanilla lines on the master, so we can see what you changed?

Reply:
Pasted vanilla lines in #1181, so, please, check it in tortoise

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/74625015-b6dc8300-515b-11ea-8476-dc5128285f17.png)

</details>